src/pivot-table/hooks/use-head-cell-dim.ts

```diff
@@ -1,22 +1,14 @@
-import { useEffect, useMemo, useState } from "react";
+import type { stardust } from "@nebula.js/stardust";
+import { useState } from "react";
 
 interface UseHeadCellDim {
-  open: boolean;
+  interactions: stardust.Interactions;
 }
 
-export const useHeadCellDim = ({ open }: UseHeadCellDim) => {
-  const [isHovered, setIsHovered] = useState(false);
+export const useHeadCellDim = ({ interactions }: UseHeadCellDim) => {
```

Comment:
Does your changes takes care of active state? 
hover state should be 3% and active state is 5% darker

Reply:
Currently no, it's roughly 7% for both hover and active. I will update the util repo so that its ~3% and ~5%. But while testing that on my screen I cannot see that color difference with only 3%, unless it goes from white to darker. 

Something like a purple color will most likely need UX calibrated screens to see any difference. But I'll stick with the values they provided anyway.

Good find!